vtr_flow/scripts/run_vtr_flow.py

```diff
@@ -488,9 +488,14 @@ def vtr_command_main(arg_list, prog=None):
 
     finally:
         seconds = datetime.now() - start
+        with open(temp_dir/"vpr.out","r") as fpmem:
+            for line in fpmem.readlines():
+                if "Maximum resident set size" in line:
+                    mem_usage = int(line.split()[-1])//1024
```

Comment:
See if you need to use the function ``format_human_readable_memory`` to format the mem values: https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/8861b1155a5efaa5f8c181063ea36173e9427644/vtr_flow/scripts/run_vtr_task.py#L439